newsletter-draft.md

```diff
@@ -0,0 +1,19 @@
+# Best Newsletter Ever!
+
+This introduction will tell our readers what this newsletter is about and help them orient themselves within the document.
+
+## Section 1
+
+This is our primary content section and will be the section that most people read. Make sure to get people hooked here and include a strong call to action
+
+## Section 2
+
+This secondary information will be less likely to be read, but will be more important for more engaged donors. Write a story about one of our donors to help them know how much their contributions mean to us.
+
+## Section 3
```

Comment:
Do we want to move this section up to the second position so that we get more donors to our events? I think that's more important than donor stories.